sdks/python/test-suites/dataflow/common.gradle

```diff
@@ -364,3 +364,34 @@ task validatesContainer() {
     }
   }
 }
+
+def tensorRTTests = tasks.create("tensorRTtests") {
+  dependsOn 'installGcpTest'
+  dependsOn ':sdks:python:sdist'
+ doLast {
+  def testOpts = basicPytestOpts
+  def argMap = [
+    "runner": "DataflowRunner",
+    "machine_type":"n1-standard-4",
+    "sdk_container_image": "us.gcr.io/apache-beam-testing/yeandy/apache_beam_tensorrt:latest",
```

Comment:
Should we make a push a copy of `us.gcr.io/apache-beam-testing/yeandy/apache_beam_tensorrt:latest` and to a path not under `yeandy`?